src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -18,12 +18,9 @@ error_reporting(-1);
 if (PHP_VERSION_ID >= 70200) {
     // PHPUnit 6 is required for PHP 7.2+
     $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '6.3';
-} elseif (PHP_VERSION_ID >= 50600) {
```

Comment:
Should be reverted, the bridge remains at 5.3

Reply:
done